### DISCUSSION TITLE
Elusive target tile replaces Freelancer tile

### DISCUSSION THREAD

**Discussion by NekoXIII** [thepeacockproject/Peacock#360] [Ideas]
A `true` / `false` option in options.ini to have the Elusive target tile which can be found in the Career tab replace the Freelancer tile on the main menu.

Mostly just QOL, I've requested a plugin by rdil with this same functionality, but it'd be nice to have it as a base option as well.
Freelancer can be accessed through the Game Modes tab, allowing it to still be played if the option were set to true.

Having it set to false by default seems like the best choice, as people are used to having the freelancer tile there.

**Comment by AnthonyFuller** [comment]
We'd just be bloating the options file. We're trying to strip down on the amount of options.

**Comment by RDIL** [comment]
Here's a version of the dashboard that does this:

[EiderDashboard.json](https://github.com/thepeacockproject/Peacock/files/13491058/EiderDashboard.json)

Feel free to use in a plugin.
